lawnchair/src/app/lawnchair/nexuslauncher/OverlayCallbackImpl.kt

```diff
@@ -81,8 +79,6 @@ class OverlayCallbackImpl(private val mLauncher: LawnchairLauncher) : LauncherOv
         mClient.onStop()
     }
 
-    override fun onActivitySaveInstanceState(activity: Activity, bundle: Bundle?) = Unit
-
```

Comment:
Does this need to be added back?

Reply:
Yeah all the overrides should be on the new file as well, I missed this one on my review.

Reply:
d0fbd9dc7023b2de24020c91977636fa8be6c34e

Reply:
I know why I removed this, the reason:

```
e: /home/runner/work/lawnchair/lawnchair/lawnchair/src/app/lawnchair/nexuslauncher/OverlayCallbackImpl.kt: (87, 5): 'onActivitySaveInstanceState' overrides nothing

FAILURE: Build completed with 2 failures.

> Task :compileLawnWithQuickstepDebugKotlin FAILED
```

Reply:
3c873a3dd77cdeb526652f2be652b7209c46f1ce

Reply:
Nice, yeah it looks like the wrong param was the issue.